composer.json

```diff
@@ -32,6 +32,7 @@
     "require-dev": {
         "knplabs/knp-menu-bundle": "^2.1.1",
         "simplethings/entity-audit-bundle": "~0.1",
+        "sllh/php-cs-fixer-styleci-bridge": "^2.0",
```

Comment:
> An extra commit is quite enough IMO, especially for a change that small.

This is not a matter of taste. :-)

https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/CONTRIBUTING.md#the-content

Several commit if this is more or less related to the PR, why not. For the rest, it would be great to have separated PR later.